core/src/main/resources/db/migration/V29__add_campaign_deltas.sql

```diff
@@ -0,0 +1,32 @@
+ALTER TABLE `Campaign`
+ADD COLUMN `status` ENUM('Draft', 'InPreparation', 'Active', 'Failed', 'Finished') NOT NULL,
```

Comment:
Have you tested this migration with existing data? Surely the `NOT NULL` constraint will fail for the existing ones, or is it that I don't understand databases?